aws/cft.yml

```diff
@@ -0,0 +1,141 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
Can we name this something more descriptive than `cft.yml`? perhaps `<project-root>/cfn/conjur.yml` or something more semantic?

Reply:
Yes, will do.

Reply:
Maybe I'm prone to over-explanation here because of unfamiliarity, but I would try harder to make the file names and directory structure descriptive. ie `<project-root>/aws-cloudfoundation/conjur-template.yml`